### PR TITLE
Add social list

### PR DESCRIPTION
**Pull request by ghost**
### What's new?

1. Created a `.social-list` component because this component can also be used in isolation from the footer
2. The `.social-list` component itself has no outer geometry, so I added a `.footer__social-list` mix to it
3. Updated styles. Aligning icons with flexbox, creating some variables and making neat code

### CODE REVIEW

File: src/styles/blocks/social-list.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.social-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.social-list__link {
+    display: flex;
```
Comment (MeFoDy) [inline src/styles/blocks/social-list.css]: Please, make it `inline-flex`. It's better when the link has no extra width on mouse events.

File: src/styles/blocks/social-list.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    background-size: contain;
+    content: '';
+}
+
+.social-list__link--twitter::before {
+    --social-icon: url('/images/twitter-logo.svg');
+}
+
+.social-list__link--github::before {
+    --social-icon: url('/images/github-logo.svg');
```
Comment (MeFoDy) [inline src/styles/blocks/social-list.css:29]: I have a dark theme on a website. Can you also fix it?
https://github.com/MeFoDy/mefody.dev/blob/612e9845c275f5307600c8f1f20b7620d73b045a/src/styles/dark.css#L37

**Review by MeFoDy — CHANGES_REQUESTED** [review]
Thank you for your PR.
Can you check my comments, please?

**Comment by ghost** [comment]
Thank you very much for the answer. I corrected what you said. I'm sorry I missed the dark mode...

**Review by MeFoDy — APPROVED** [review]

**Comment by MeFoDy** [comment]
Thank you!

**Pull request merged**